acquia/acquia_cms
# ACMS-1410: Upgrade ACMS modules to make Drupal 10 compatible.

**Motivation**
Fixes #ACMS-1410

**Proposed changes**
Upgrade ACMS modules to make Drupal 10 compatible.

**Alternatives considered**
NA

**Testing steps**
- Use this branch
- Install site 
- Verify working fine
- Execute rector is should not show any warning/error

**Merge requirements**
